### DISCUSSION TITLE
Follow-up on pvlib PVPMC training session

### DISCUSSION THREAD

**Discussion by AdamRJensen** [PVSC-Python-Tutorials/PVPMC_2023#18] [Discussion]
I think it would be great to jot down a few notes on how the pvlib hackathon went. Hopefully, we can pass on some useful information to @mikofski and @shirubana for PVSC.

**General comments**
- The attendance was high, with more than 40 participants!
- Overall I [Adam] think it went exceptionally well with good user engagement
- There was generally a large number of questions during the session - three "helpers" (@kandersolar, @cwhanse, and myself) to 40 participants worked well
- There was a good split between attendees working on their own projects and projects from the suggested list

**Practical**
- Would be nice with more power outlets/cords
- @kandersolar mentioned that it would have been nice with a survey to the users

**Google Colab**
- A 5-minute introduction to Google Colab seemed sufficient
- Google Colab worked effortlessly and can be highly recommended for future tutorials
- Google Colab doesn't work with local files - keep this in mind!

@cwhanse and @kandersolar please add your own feedback either as comments or by editing this post.

**Comment by cwhanse** [comment]
We should do each of the suggested projects ourselves.

A prize for people who find bugs - something simple.

Bring the stickers!

**Reply by mikofski** [reply]
This is great news and really useful feedback. Thanks!

**Comment by mikofski** [comment]
For future reference, I have been able to use local files, but it wasn't immediately obvious: https://colab.research.google.com/notebooks/io.ipynb
